lib/src/common/syntax.dart

```diff
@@ -1,28 +1,34 @@
 part of guinness2;
 
 void beforeEach(Function fn, {int priority: 0}) =>
```

Comment:
It's probably a good idea to deprecate or remove this priority parameter, since `test` doesn't support it.